code_generation/quantities.py

```diff
@@ -68,6 +68,14 @@ def get_shifts(self, scope):
 Muon_mass = Quantity("Muon_mass")
 Muon_iso = Quantity("Muon_pfRelIso04_all")
 
+electron_veto_mask = Quantity("electron_veto_mask")
+electron_veto_flag = Quantity("extraelec_veto")
+Electron_pt = Quantity("Electron_pt")
+Electron_eta = Quantity("Electron_eta")
+Electron_phi = Quantity("Electron_phi")
+Electron_mass = Quantity("Electron_mass")
+Electron_iso = Quantity("Electron_pfRelIso03_all")
```

Comment:
here, its that way so the correct input is used, its not the quantity that is written into the ntuple, but the mapping from NanoAOD to our framework

Reply:
currently, the MuomIsoFilter is implement exactly the same way, with 

```
MuonIsoFilter = Producer(
    name="MuonIsoFilter",
    call="physicsobject::muon::FilterIsolation({df}, {output}, {input}, {muon_iso_cut})",
    input=[q.Muon_iso],
    output=[],
    scopes=["global"],
)

Muon_iso = Quantity("Muon_pfRelIso04_all")
```

Reply:
Ah sorry, my bad. Apparently I had that once, but for some reason changed it before committing. This is a remainder of it: https://github.com/KIT-CMS/CROWN/blob/main/config/config.py#L21
I don't remember my reasoning at that time. What do you think is the better approach? Currently, I don't see a benefit from using it as quantity.

Reply:
I think the it is currently is a bit inconsistent in the way, that there is a difference between quantities, that are included in the NanoAOD and quantities, that are calculated from our side. Of course writing out something like `Muon_iso-->(Muon_pfRelIso04_all)` does not make any sense, since this is an `RVec` and not a flat ntuple value, so we MUST not use it as final output. Maybe it would be worth to split quantities further into two types, the ones from NanoAOD which are mainly used as input for producers and are not meant for the final ntuple, and flat quantities like `pt_1`, which are calculated and also written out.

Reply:
Yes, in principle, we could achieve this by hardcoding quantities that exist in nanoAOD. I think there is no reason to include them in the quantity machinery.

Reply:
so we do an inheritant class of the quantities, where something like shifting is not possible, do we dont even run into problems at that front. Something like `AODInputQuantity` ?

Reply:
But i would put this in a separate PR